src/redisearch_rs/tools/shadow_tester/src/args.rs

```diff
@@ -0,0 +1,20 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+#[derive(Parser)]
+#[command(version, about)]
+pub struct Options {
+    /// Path to the baseline .so file. The responses from this module will be considered the source
+    /// of truth.
+    #[arg(short, long)]
```

Comment:
I think it doesn't add much value to have short (-b,-c) versions of these flags. They will only cause confusion

Reply:
Fair point. WIll change